gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStorageStatistics.java

```diff
@@ -26,7 +26,7 @@
 public class GhfsStorageStatistics extends StorageStatisticsFromIOStatistics {
 
   /** {@value} The key that stores all the registered metrics */
-  public static final String NAME = "GhfsStorageStatistics";
+  public static final String NAME = "GhfsFileSystemBasedStorageStatistics";
```

Comment:
Our spark plugin will not take this name and take "GhfsGlobalStorageStatistics" which stores the per executor metrics.